stl/inc/chrono

```diff
@@ -278,6 +279,13 @@ namespace chrono {
     };
 } // namespace chrono
 
+template <class _Rep, class _Period>
+_INLINE_VAR constexpr bool _Is_trivially_swappable_v<chrono::duration<_Rep, _Period>> = _Is_trivially_swappable_v<_Rep>;
+
+template <class _Clock, class _Duration>
+_INLINE_VAR constexpr bool _Is_trivially_swappable_v<chrono::time_point<_Clock, _Duration>> =
+    _Is_trivially_swappable_v<_Duration>;

```

Comment:
I'm mildly concerned that this relationship might not hold for program-defined specializations of `duration` and `time_point`. Should we use the `_From_primary` technique from `iterator_traits` to constrain this to library-defined specializations only?

Reply:
I think that this is fine as-is - I can't imagine why a program would ever attempt to specialize `duration` and `time_point` (unlike `hash` or `less`, or even `allocator`), and even if so, how they could be non-trivially-swappable differently from their `_Duration` and `_Rep`. Theoretically it is possible, but I believe we will never ever see it in practice.

The current PR handles the case where `duration` and `time_point` are the STL's primary templates, but a user-defined representation type (possibly a bignum) is being used - that is also very rare but within the realm of possibility that we should attempt to handle.

Reply:
As long as we're all clear that I get to say "I told you so" if there's ever a bug report, I'm ok with this =)